tests/resources/test_issue.py

```diff
@@ -161,21 +148,21 @@ def test_create_issues_one_failure(self):
                 "description": "Some new issue for test",
                 "issuetype": {"name": "Bug"},
```

Comment:
(non-blocking) while you are here can I ask you to move this up a bit to match the format of the other entries, i.e. where the `issuetype` is right after `project` ? Or maybe it makes sense for the `summary` to be first as it is explaining the test case?

Feel free to ignore.

Reply:
Done.